src/rules/at-function-named-arguments/index.js

```diff
@@ -0,0 +1,125 @@
+import { utils } from "stylelint";
+import {
+  namespace,
+  optionsHaveIgnored,
+  isNativeCssFunction
+} from "../../utils";
+import valueParser from "postcss-value-parser";
+
+export const ruleName = namespace("at-function-named-arguments");
+
+export const messages = utils.ruleMessages(ruleName, {
+  expected: "Expected a named parameter to be used in function call",
+  rejected: "Unexpected a named parameter in function call",
+  rejectedSingle:
```

Comment:
I had another look at the code and I noticed that `rejectedSingle` does not seem to be used anywhere. It's also in the other rule.